### ISSUE TITLE
peerDependency on webpack broken for webpack 5 since v3.2.0

### ISSUE THREAD

**Issue by jorrit** [shellscape/webpack-manifest-plugin#270]
In commit 79987fcd1314286a33441d306e7b85b5445056b2, which is included in 3.2.0, the peerDependency argument of webpack got changed from `>=4.44.2` to `^4.44.2`. This means that from 3.2.0 on, Webpack 5 is not allowed as a peer dependency.

- Manifest Plugin Version: 3.2.0
- Webpack Version: 5.46.0
- Operating System (or Browser): Win 10
- Node Version: 14
- Link to reproduction _(⚠️ read below)_:

### Repro

With NPM 7:

1. cd to some empty directory
2. `npm install webpack@5`
3. `npm install webpack-manifest-plugin`

### Expected Behavior

Successful installation

### Actual Behavior

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: undefined@undefined
npm ERR! Found: webpack@5.46.0
npm ERR! node_modules/webpack
npm ERR!   webpack@"^5.46.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer webpack@"^4.44.2" from webpack-manifest-plugin@3.2.0
npm ERR! node_modules/webpack-manifest-plugin
npm ERR!   webpack-manifest-plugin@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

### Additional information

Running `npm install webpack-manifest-plugin@3.1.1` works fine.

**Comment by ali-hosseini-fs** [comment]
When do you want to publish @jorrit s PR to npm?
I updated npm/node version on our project and because of this problem on your package I can't continue my work on it!
I will appreciate if you can publish it as soon as possible.

**Comment by shellscape** [comment]
@ali-hosseini-fs when I'm damn well ready to. there isn't a maintainer alive who appreciates those kinds of replies. stop it.

**Comment by shellscape** [comment]
Closed by #271 _yesterday_

**Comment by yura3d** [comment]
Just updated to 4.0.0 and still see warning for incorrect webpack version (^4.44.2) as peer dependency. Opened package.json and saw:
https://github.com/shellscape/webpack-manifest-plugin/blob/5ff683260cb6b7f48ad8a0f02faebe89a63d964e/package.json#L2-L3
https://github.com/shellscape/webpack-manifest-plugin/blob/5ff683260cb6b7f48ad8a0f02faebe89a63d964e/package.json#L38-L40
